src/org/zaproxy/zap/extension/domxss/TestDomXSS.java

```diff
@@ -86,6 +95,8 @@
 
     private static ProxyServer proxy = null;
     private static int proxyPort = -1;
+    
+    private static WebDriverWrapper ffDriver;
```

Comment:
Without static, this one is just for the instance. May I suggest `driver`? It would require less renames in the other methods.